src/Playgrounds/CosmosEndToEnd/CosmosEndToEnd.ApiService/Program.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Azure.Cosmos;
+using Newtonsoft.Json;
+
+var sessionId = Guid.NewGuid().ToString();
```

Comment:
What's this for.

Reply:
It's an artificial partition key. Cosmos containers require all rows to have an ID and partition ID. The idea (as I understand it) is that if you write data that is related (e.g. a bunch of child records) you'll probably want them to come from the same partition unless it's a very large set.

In the context of this sample, I'm just creating a session ID that I used for each entry I create. My thinking was that if we use this playground against a live database being able to track the session a particular entry was created in might be useful, and it probably makes sense for it to be the partition key.

I can remove it and make the Id field the partition key if it causes too much confusion.